mteb/tasks/STS/rus/RUParaPhraserSTS.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskSTS import AbsTaskSTS
+
+
+class RUParaPhraserSTS(AbsTaskSTS):
+    metadata = TaskMetadata(
+        name="RUParaPhraserSTS",
+        dataset={
+            "path": "merionum/ru_paraphraser",
+            "revision": "43265056790b8f7c59e0139acb4be0a8dad2c8f4",
+        },
+        description="Human-annotated STS dataset of Korean reviews, news, and spoken word sets. Part of the Korean Language Understanding Evaluation (KLUE).",
+        reference="https://arxiv.org/abs/2105.09680",
+        type="STS",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["rus-Cyrl"],
+        main_score="cosine_spearman",
+        date=("2009-01-01", "2019-01-01"),  # rough estimate,
+        form=["written"],
+        domains=["News"],
+        task_subtypes=[],
+        license="MIT",
+        socioeconomic_status="high",
+        annotations_creators="human-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+@inproceedings{gudkov-etal-2020-automatically,
```

Comment:
nit: Should we cite this as well?

```
@inproceedings{pivovarova2017paraphraser,
  title={ParaPhraser: Russian paraphrase corpus and shared task},
  author={Pivovarova, Lidia and Pronoza, Ekaterina and Yagunova, Elena and Pronoza, Anton},
  booktitle={Conference on artificial intelligence and natural language},
  pages={211--225},
  year={2017},
  organization={Springer}
}
```

Reply:
Thanks @KranthiGV, happy to do that, I am just not sure what would be the proper way of doing so, as I didn't really see it being used elsewhere.

Would you mind helping me out with that?

Reply:
You can use the bibtex entry above and add it similar to `InternationalCitizenshipQuestionsLegalBenchClassification` which has two citations as well.

Reply:
Thanks @KranthiGV, should be resolved now.